app/models/financialDetails/TransactionItem.scala

```diff
@@ -47,7 +47,7 @@ trait TransactionItem {
   }
 
   // TODO: duplicate logic, in scope of => https://jira.tools.tax.service.gov.uk/browse/MISUV-8557
-  def getChargeTypeKey(reviewAndReconcileEnabled: Boolean = false, penaltiesEnabled: Boolean = false): String =
+  def getChargeTypeKey: String =
```

Comment:
Just wondering if we can move this under ChargeNameLangHelper as a separate method?

Reply:
It is used in quite a few places, and would need to be called like this in the views:
ChargeNameLangHelper.getChargeTypeKey(viewModel.chargeItem.transactionType, viewModel.chargeItem.subTransactionType)
I'm not sure if this is worth it

Reply:
as discussed: will be covered by 
https://jira.tools.tax.service.gov.uk/browse/MISUV-8557